reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+              p result
+            rescue SwaggerClient::ApiError => e
+              puts "Exception when calling GetApi->admin_draft_ordersid_json_get: #{e}"
+            end
+            
+  
+            
+            
+    
+    delete:
+      description: Remove a draft order
+      tags:
+        - Delete
+      parameters:
+        - name: id
+          in: path
+          description: draft order id
+          required: true
+          type: integer
+          format: int64
+      responses:
+        '200':
+          description: Deleted draft order
+          schema:
+            $ref: '#/definitions/DraftOrder'
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X DELETE -H "Authorization: Basic YOUR-API-KEY=" "https://your_store_domain.myshopify.com/admin/draft_orders/57777542.json"
+      
+  /admin/draft_orders/count.json:
+    get:
+      summary: Return count of draft orders
+      description: This endpoint enables you to return a count of all draft orders.    
+      parameters:
+       - $ref: '#/parameters/fields'
+       - $ref: '#/parameters/limit'
+       - $ref: '#/parameters/page'
+       - $ref: '#/parameters/since_id'
+       - $ref: '#/parameters/status'
+       - $ref: '#/parameters/ids'
+      tags:
+        - Get
+      responses:
+        '200':
+           description: Count of draft orders
+           $ref: '#/responses/Integer'
+
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X GET -H "Authorization: Basic YOUR-API-KEY=" "https://your_store_domain.myshopify.com/admin/draft_orders/count.json"
+      
+        - lang: Ruby
+          source: |
+            # load the gem
+            require 'swagger_client'
+            
+            api_instance = SwaggerClient::GetApi.new
+            
+            begin
+              result = api_instance.admin_draft_orders_count_json_get
+              p result
+            rescue SwaggerClient::ApiError => e
+              puts "Exception when calling GetApi->admin_draft_orders_count_json_get: #{e}"
+            end
+
+
+
+  '/admin/draft_orders/#{id}/send_invoice.json':
+    post:
+      summary: Create an invoice for a draft order
+      description: You can use this endpoint to send your customer an invoice via email for thier draft order.  The invoice contains a url link, enabling the customer to pay for the item using web checkout.  Additionally, you can send a `draft_order_invoice` object in the body of the request to override any of the default email parameters for the invoice.  
+      tags:
+        - Create
+      parameters:
+        - name: draft_order_invoice
+          in: body
+          description: draft_order_invoice to post.
+          schema:
+            allOf:
+              - $ref: '#/definitions/draft_order_invoice'
+      
+        - name: id
+          in: path
+          required: true
+          type: integer
+
+
+      responses:
+        '201':
```

Comment:
I'm not sure if this is 201, did you test? Also you can likely get a 422 for bad parameters, or 404 for wrong draft order id that cant be found